src/docs/product/integrations/integration-platform/webhooks.mdx

```diff
@@ -115,827 +120,19 @@ actor
 ```
 
 <Note>
-
-You cannot set up a custom payload. Instead, we suggest setting up a server/service that proxies our webhook into a request that fits your use case.
-
+You cannot set up a custom payload. Instead, we suggest setting up a server or service that proxies our webhook into a request that fits your use case.
 </Note>
```

Comment:
TBH I think this note can just be removed. I don't think webhooks are usually customizable, and if they are its a feature.

If anything, you can call out that certain elements of the `data` blob we talk about earlier can be modified with [UI Components](#)